src/app/admin/user-panel/user-edit-roles/edit-user-roles.component.scss

```diff
@@ -0,0 +1,29 @@
+.edit-dialog {
+  h2 {
```

Comment:
in the future, this is the kind of style that should be standardized. or just update the material theme, probably not worth the time now.

Reply:
Good point